src/index.html

```diff
@@ -11,12 +11,107 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          class="header__logo--link"
```

Comment:
```suggestion
          class="header__logo-link"
```
it's not modificator - it's just an element of `header` block. One BEM element can be nested inside another BEM element